packages/forms/src/UIForm/fields/CheckBox/SimpleCheckBox.component.js

```diff
@@ -23,11 +23,11 @@ if (process.env.NODE_ENV !== 'production') {
 	SimpleCheckBox.propTypes = {
 		id: PropTypes.string,
 		label: PropTypes.string,
-		onChange: PropTypes.func,
+		onChange: PropTypes.func.isRequired,
 		schema: PropTypes.shape({
 			autoFocus: PropTypes.bool,
 			disabled: PropTypes.bool,
-		}),
+		}).isRequired,
```

Comment:
schema is required and none of its component is, isn't it more safe then to rely on a default props set to an empty literral object ?